region/maridia/inner-pink/Crab Shaft.json

```diff
@@ -1467,6 +1467,40 @@
       },
       "flashSuitChecked": true
     },
+    {
+      "link": [2, 2],
+      "name": "R-Mode Spark Interrupt (Gain Blue Suit)",
+      "requires": [
+        {"obstaclesCleared": ["R-Mode"]},
+        "Gravity",
+        {"or": [
+          "h_CrystalFlashForReserveEnergy",
+          {"and": [
+            "h_RModeCanRefillReserves",
+            {"or": [
+              {"and": [
+                {"resourceMissingAtMost": [{"type": "PowerBomb", "count": 2}]},
+                {"partialRefill": {"type": "ReserveEnergy", "limit": 40}}
+              ]},
+              {"and": [
+                {"resourceMissingAtMost": [{"type": "PowerBomb", "count": 0}]},
+                {"partialRefill": {"type": "ReserveEnergy", "limit": 80}}
```

Comment:
Does this mean, 
- put 60 energy in reserves
- activate reserves
- put 20 energy in main energy?

Reply:
Scisers are 0% Nothing so the "interrupt crab" can be included in the farming output (but not ammo leniency).

Supposed to be:
- If you're full on PBs you will very likely get large energy from the first three Scisers. (Small/large ratio is highly weighted toward large. Similar to Crab Maze.)
- If you aren't you can afford to be missing 2 PBs - get them back from Scisers - and still have one left to get energy from.

Then after you trigger, you can kill the last crab for (likely) another large energy.